cloudigrade/account/tasks.py

```diff
@@ -161,8 +241,9 @@ def run_inspection_cluster(messages, cloud='aws'):
     Run task definition for "houndigrade" on the cluster.
 
     Args:
-        messages (list): A list of dictionary items containing
-            the ami_id and volume_id to be inspected.
+        messages (list): A list of dictionary items containing meta-data (arn,
+            source_region, ami_id, customer_snapshot_id, snapshot_copy_id,
```

Comment:
I think this comment is a leftover from an earlier stage of refactoring. It looks like `messages` is back down to just `ami_id` and `volume_id`.